packages/vats/src/types.d.ts

```diff
@@ -106,4 +106,139 @@ export type BridgeManager = {
   ) => ScopedBridgeManager;
 };
 
-export type * from './ibc.js';
+export type IBCPortID = string;
```

Comment:
why did you move these from `ibc.js`? I suppose to get .ts syntax. I expect there's a better place for these but I can't think of a good suggestion atm

Reply:
Yes for TS syntax, plus this ballooned to quite a few lines at the top of the file.

I attempted `ibc.d.ts` in the same directory but this caused issues with things looking for exports from `ibc.js`. I also did not observe a pattern of multiple .d.ts files in a single package in the repo